roles/edpm_ovn/tasks/configure.yml

```diff
@@ -14,6 +14,18 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Apply ovsdb configuration from configMaps
+  block:
+  - name: Load ovsdb configuration from file
+    ansible.builtin.include_vars:
+      file: "{{ edpm_ovn_config_src }}/ovsdb-config"
+      name: edpm_ovn_configmap_facts
+    delegate_to: localhost
```

Comment:
as a fail safe could also add so it works without config map
when: "'{{ edpm_ovn_config_src }}/ovsdb-config' is file"

Reply:
Great. Ansible discourage jinja in conditionals, so I will handle it a bit differently, but overall in the direction you suggested.